### PR TITLE
Enable S3 ingestion

### PR DESCRIPTION
**Pull request by dtitov**
### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:
Enables S3 ignestion.

### Changes made:
Ingestion service is now using s3-enabled image (`nbisweden/ega-base:s3`).

### Related issues:
https://github.com/NBISweden/LocalEGA-deploy-swarm/issues/16

### CODE REVIEW

File: buildSrc/src/main/java/se/nbis/lega/deployment/lega/priv/CreateConfIniConfigurationTask.java [hunk 1]
```diff
@@ -16,26 +16,28 @@ public void run() throws IOException {
...
+                + "access_key = %s\n" + "secret_key = %s\n" + "#region = lega\n" + "\n"
                 + "[outgestion]\n" + "# Just for test\n"
                 + "keyserver_endpoint = http://keys:8080/keys/retrieve/%%s/private/bin?idFormat=hex\n"
-                + System.lineSeparator() + "## Connecting to Local EGA\n" + "[broker]\n"
-                + "host = mq\n" + "connection_attempts = 30\n" + "# delay in seconds\n"
-                + "retry_delay = 10\n" + System.lineSeparator() + "[postgres]\n" + "host = db\n"
-                + "port = 5432\n" + "user = lega_in\n" + "password = %s\n" + "database = lega\n"
-                + "try = 30\n" + "sslmode = require\n" + System.lineSeparator() + "[eureka]\n"
-                + "endpoint = http://cega-eureka:8761", s3AccessKey, s3SecretKey, postgresPassword),
+                + "\n" + "## Connecting to Local EGA\n" + "[broker]\n" + "host = mq\n"
```
Comment (amgadhanafy) [inline buildSrc/src/main/java/se/nbis/lega/deployment/lega/priv/CreateConfIniConfigurationTask.java:35]: why not `System.lineSeparator() `

**Review by amgadhanafy — COMMENTED** [review]
why not `System.lineSeparator() `

File: buildSrc/src/main/java/se/nbis/lega/deployment/lega/priv/CreateConfIniConfigurationTask.java (hunk 1 above)
Comment (dtitov) [reply buildSrc/src/main/java/se/nbis/lega/deployment/lega/priv/CreateConfIniConfigurationTask.java:35]: To be able to make use of this feature: 
Also, this is how IntelliJ automatically handles newlines.

Comment (amgadhanafy) [reply buildSrc/src/main/java/se/nbis/lega/deployment/lega/priv/CreateConfIniConfigurationTask.java:35]: not happy with it but ok

**Review by amgadhanafy — APPROVED** [review]

**Pull request merged**